apps/fields/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
@JoeChapman @joefitter @gxxm @easternbloc How do you guys feel about the structure being '/apps/fields/.....' or do you think it should be 'apps/complaints/fields/....'?